feature/notification/build.gradle

```diff
@@ -0,0 +1,53 @@
+import dependencies.Dep
+
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-kapt'
+apply plugin: 'androidx.navigation.safeargs'
+
+apply from: rootProject.file('gradle/android.gradle')
+android.dataBinding.enabled = true
+
+dependencies {
+    implementation project(':feature:system')
+
+    implementation project(":data:repository")
+
+    implementation project(':corecomponent:androidcomponent')
+    implementation project(':ext:log')
+
+    implementation Dep.Kotlin.stdlibJvm
+    implementation Dep.Kotlin.coroutines
+    implementation Dep.Kotlin.coroutinesPlayServices
+    implementation Dep.Kotlin.androidCoroutinesDispatcher
+
+    implementation Dep.AndroidX.appCompat
+    implementation Dep.AndroidX.constraint
+    implementation Dep.AndroidX.recyclerView
+    implementation Dep.AndroidX.emoji
+    implementation Dep.AndroidX.design
+    implementation Dep.AndroidX.Work.runtimeKtx
+
+    implementation Dep.Timber.android
+    implementation Dep.Firebase.messaging
+
+    implementation Dep.Dagger.core
+    implementation Dep.Dagger.androidSupport
+    kapt Dep.Dagger.compiler
+    kapt Dep.Dagger.androidProcessor
+    compileOnly Dep.Dagger.assistedInjectAnnotations
+    kapt Dep.Dagger.assistedInjectProcessor
+
+    implementation Dep.Groupie.groupie
+    implementation Dep.Groupie.databinding
```

Comment:
I think we can remove UI libraries. 🙇 
    implementation Dep.Groupie.groupie
    implementation Dep.Groupie.databinding
    implementation Dep.AndroidX.recyclerView
    implementation Dep.Coil.coil
    implementation Dep.AndroidX.emoji
    implementation Dep.AndroidX.design